.github/ISSUE_TEMPLATE/Standard-Investigation-Issue-Template.md

```diff
@@ -0,0 +1,39 @@
+---
+name:  Investigation Issue
+about: Use this template for all Investigations
+labels: Investigation
+---
+# Story
+## Inspiration
+< What inspired this investigation >
+
+## Value of information acquired
+< What is the potential gain from the investigation outcome (customer dissatisfaction, loss of time, money, etc. >
```

Comment:
I'm not sure that "customer dissatisfaction", "loss of time" or "loss of money" are gains 😛

Reply:
would you believe reverse psychology? Will reverse them!